apps/condo/domains/user/schema/RegisterNewServiceUserService.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * Generated by `createservice user.RegisterNewServiceUserService`
+ */
+
+const { GQLCustomSchema, getById} = require('@core/keystone/schema')
+const access = require('@condo/domains/user/access/RegisterNewServiceUserService')
+const {STAFF, MIN_PASSWORD_LENGTH} = require("@condo/domains/user/constants/common");
+const {
+    MIN_PASSWORD_LENGTH_ERROR
+} = require("@condo/domains/user/constants/errors");
+const { ensureNotExists } = require('@condo/domains/user/utils/ensureNotExists')
+
+
+const RegisterNewServiceUserService = new GQLCustomSchema('RegisterNewServiceUserService', {
+    types: [
+        {
+            access: true,
+            type: 'input RegisterNewServiceUserInput {  dv: Int!, sender: SenderFieldInput!, name: String!, email: String!, password: String! meta: JSON }',
+        },
+    ],
+    
+    mutations: [
+        {
+            access: access.canRegisterNewServiceUser,
+            schema: 'registerNewServiceUser(data: RegisterNewServiceUserInput!): User',
+            resolver: async (parent, args, context) => {
+                const { data } = args
+                const { ...restUserData } = data
+                const userData = {
+                    ...restUserData,
+                    type: STAFF,
+                }
```

Comment:
why not simply make `const userData = { ...args.data, type: STAFF }` I think using three variables here is redundant